_config.yml

```diff
@@ -105,7 +86,54 @@ source_releases:
 
 stable_releases:
   -
-      name: "Apache Flink 1.7.2 only"
+      name: "Apache Flink 1.8.0"
+      version_short: 1.8
+      scala_211:
+          id: "180-download_211"
+          url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.8.0/flink-1.8.0-bin-scala_2.11.tgz"
+          asc_url: "https://www.apache.org/dist/flink/flink-1.8.0/flink-1.8.0-bin-scala_2.11.tgz.asc"
+          sha512_url: "https://www.apache.org/dist/flink/flink-1.8.0/flink-1.8.0-bin-scala_2.11.tgz.sha512"
+      scala_212:
+          id: "180-download_212"
+          url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.8.0/flink-1.8.0-bin-scala_2.12.tgz"
+          asc_url: "https://www.apache.org/dist/flink/flink-1.8.0/flink-1.8.0-bin-scala_2.12.tgz.asc"
+          sha512_url: "https://www.apache.org/dist/flink/flink-1.8.0/flink-1.8.0-bin-scala_2.12.tgz.sha512"
+      optional_components:
+        -
+          name: "Pre-bundled Hadoop 2.8.0"
+          category: "Pre-bundled Hadoop"
+          scala_dependent: false
+          id: 180-bundled-hadoop
+          url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.7.2/flink-avro-1.7.2.jar
+          asc_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.7.2/flink-avro-1.7.2.jar.asc
+          sha_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.7.2/flink-avro-1.7.2.jar.sha1
+        -
+          name: "Pre-bundled Hadoop 2.7.0"
+          category: "Pre-bundled Hadoop"
+          scala_dependent: false
+          id: 180-bundled-hadoop
+          url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.7.2/flink-avro-1.7.2.jar
+          asc_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.7.2/flink-avro-1.7.2.jar.asc
+          sha_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.7.2/flink-avro-1.7.2.jar.sha1
+        -
+          name: "Avro SQL Format"
+          category: "SQL Formats"
+          scala_dependent: false
+          id: 180-sql-format-avro
+          url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.8.0/flink-avro-1.8.0.jar
+          asc_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.8.0/flink-avro-1.8.0.jar.asc
+          sha_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.8.0/flink-avro-1.8.0.jar.sha1
+        -
+          name: "JSON SQL Format"
```

Comment:
We forgot to add the new CSV format here. Could you add it?

Reply:
done